bucket/unzip.json

```diff
@@ -2,10 +2,7 @@
     "version": "6.00",
     "description": "Unzip compression utility",
     "homepage": "http://www.info-zip.org/",
-    "license": {
-        "identifier": "Freeware",
-        "url": "http://infozip.sourceforge.net/doc/LICENSE"
-    },
```

Comment:
Do we have a standard practice/rule to leave off the url if it's a known SPDX license? The url in this case would be http://infozip.sourceforge.net/license.html found on https://spdx.org/licenses/Info-ZIP.html.

Reply:
That's what most of the manifests in the official buckets do, so I'm just following it for consistency.